app/org/apache/spark/deploy/history/SparkFSFetcher.scala

```diff
@@ -338,6 +339,7 @@ private object SparkFSFetcher {
 
   var defEventLogDir = "/system/spark-history"
   var defEventLogSizeInMb = 100d; // 100MB
+  var defSparkLogExt = "_1.snappy"
```

Comment:
Ah ok. I didn't realize that since that's not how our cluster is configured and there isn't any logic to try to read other attempts. But yes I would agree that that seems like a separate issue.